src/Plugins/ITKImageProcessing/src/ITKImageProcessing/Filters/ITKImportImageStackFilter.cpp

```diff
@@ -30,15 +30,15 @@ using namespace nx::core;
 namespace RotateSampleRefFrame
 {
 // Parameter Keys
-static inline constexpr nx::core::StringLiteral k_RotationRepresentation_Key = "rotation_representation";
-static inline constexpr nx::core::StringLiteral k_RotationAxisAngle_Key = "rotation_axis";
-static inline constexpr nx::core::StringLiteral k_RotationMatrix_Key = "rotation_matrix";
-static inline constexpr nx::core::StringLiteral k_SelectedImageGeometryPath_Key = "input_image_geometry_path";
-static inline constexpr nx::core::StringLiteral k_CreatedImageGeometry_Key = "output_image_geometry_path";
-static inline constexpr nx::core::StringLiteral k_RotateSliceBySlice_Key = "rotate_slice_by_slice";
-static inline constexpr nx::core::StringLiteral k_RemoveOriginalGeometry_Key = "remove_original_geometry";
-
-// static inline constexpr nx::core::StringLiteral k_RotatedGeometryName = ".RotatedGeometry";
+static constexpr nx::core::StringLiteral k_RotationRepresentation_Key = "rotation_representation";
+static constexpr nx::core::StringLiteral k_RotationAxisAngle_Key = "rotation_axis";
+static constexpr nx::core::StringLiteral k_RotationMatrix_Key = "rotation_matrix";
+static constexpr nx::core::StringLiteral k_SelectedImageGeometryPath_Key = "input_image_geometry_path";
+static constexpr nx::core::StringLiteral k_CreatedImageGeometry_Key = "output_image_geometry_path";
+static constexpr nx::core::StringLiteral k_RotateSliceBySlice_Key = "rotate_slice_by_slice";
+static constexpr nx::core::StringLiteral k_RemoveOriginalGeometry_Key = "remove_original_geometry";
```

Comment:
```suggestion
constexpr nx::core::StringLiteral k_RotationRepresentation_Key = "rotation_representation";
constexpr nx::core::StringLiteral k_RotationAxisAngle_Key = "rotation_axis";
constexpr nx::core::StringLiteral k_RotationMatrix_Key = "rotation_matrix";
constexpr nx::core::StringLiteral k_SelectedImageGeometryPath_Key = "input_image_geometry_path";
constexpr nx::core::StringLiteral k_CreatedImageGeometry_Key = "output_image_geometry_path";
constexpr nx::core::StringLiteral k_RotateSliceBySlice_Key = "rotate_slice_by_slice";
constexpr nx::core::StringLiteral k_RemoveOriginalGeometry_Key = "remove_original_geometry";
```
These along with the enum belong should be moved into the anonymous namespace in this file.